### DISCUSSION TITLE
请问下移动端BMP本地缓存如何关闭

### DISCUSSION THREAD

**Discussion by zhengcling** [Tencent/libpag#2207] [Q&A] [answered]
如题

**Comment by kevingpqi123** [comment]
PAG 4.3 版本才有 BMP 缓存本地的功能，默认是关闭的，这里在 API 文档中就可以看到，你指的是否是这个？
If set to true, [PAG](https://pag.art/apis/ios/Classes/PAG.html) will cache the associated rendering data into a disk file, such as the decoded image frames of video compositions. This can help reduce memory usage and improve rendering performance.

- (BOOL)useDiskCache
Declared In
PAGView.h

**Reply by zhengcling** [reply, marked as answer]
应该是微信浏览器的缓存导致的？更新bmp内容渲染未更新
